app/src/main/res/layout/activity_main.xml

```diff
@@ -6,6 +6,14 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
+    <FrameLayout
+        android:id="@+id/frame"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
```

Comment:
When I was working on my layout I ended up changing the layout_height here to 0dp so that this frame expanded to fill all of the available space within the constraint layout. What do you think about doing this here? Wonder if there are other ways to have my layout expanded fully within this frame also besides what I did.

Reply:
@brownyte